lib/internal/freelist.js

```diff
@@ -25,6 +27,11 @@ class FreeList {
     }
     return false;
   }
+
+  updateMax(max) {
```

Comment:
I don't think adding this function is needed. We can just validate and set `max` directly from `setMaxIdleHTTPParser()`.

Reply:
Fixed, thanks. I just think `max` is a 'private' field so we should access it by a public API. But validating and setting it directly is also ok.

Reply:
`FreeList` is not a public API and the properties aren't even underscore-prefixed, so I think it's fine.